src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/DeleteTaskByIdControllerTest.java

```diff
@@ -102,6 +102,45 @@ public void should_succeed_when_terminate_reason_is_completed() {
 
     }
 
+    @Test
+    public void should_succeed_and_set_termination_process_when_cancellation_process_variable_available_in_camunda() {
```

Comment:
Although this test suggests that termination process will be asserted, I'm unable to see any checks here